### ISSUE TITLE
Accessibility: Add alt text for buttons

### ISSUE THREAD

**Issue by carlycurran** [openstax/tutor#8]
## Description
Add alt text for menu button, reading breadcrumbs, reference view buttons. Need clarification on accessibility call. 

## Acceptance 
Buttons above have alt text

**Comment by carlycurran** [comment]
Added text to buttons in the UI -- should be read with jaws now. 
- menu button dropdowns
- breadcrumbs/milestones for reading and HW
- student dashboard "view book" button

**Comment by carlycurran** [comment]
@nathanstitt 

I found some more of Larissa's directions in her doc. Looks like there are still a few things missing. 
I'm sorry -- this is totally on me for not investigating more thoroughly before now. 

Reading assignment main menu: 
- calendar button not clear, says "button" and date number
- grid representation of milestones says "link" -- not clear what it goes to 
- highlighting summary page says "unlabeled 0 button" 
- Log Out menu item says "2 of 2"

**Comment by Malar-Natarajan** [comment]
Retested in Staging:
1. Menu dropdown items are readable by JAWS now
2. Breadcrumbs/milestones are readable by JAWS
3. student dashboard "view book" button by JAWS
4. Logout menu item is readable by JAWS
5. Calendar button read correctly as (Due on 11/3...whatever is displayed in the hover text)


Remaining pointers that are not working properly are recorded in a new issue
